pwiz_tools/Skyline/TestPerf/PerfCommandlineCreateImsDbTest.cs

```diff
@@ -43,55 +50,78 @@ public void CommandlineCreateImsDbPerfTest()
             TestFilesPersistent = new[] {  rawFile }; // list of files that we'd like to unzip alongside parent zipFile, and (re)use in place
             TestFilesDir = new TestFilesDir(TestContext, TestFilesZip, ".", TestFilesPersistent);
 
-            var badName = @"blorf";
-            foreach (var libName in new[]{@"_myname", string.Empty, badName}) // Try with and without explicit library name, and with an illegal name
+            var badName = @"badname";
```

Comment:
Why not calls to a sub-function? Can you try re-writing this with a sub-function instead of a loop?

Reply:
Will do